lib/GLPI/Agent/HTTP/Client.pm

```diff
@@ -553,9 +553,6 @@ sub _setSSLOptions {
 sub _KeyChain_or_KeyStore_Export {
     my ($self) = @_;
 
-    # Only MacOSX and MSWin32 are supported
-    return unless $OSNAME =~ /^darwin|MSWin32$/;
-
```

Comment:
You probably misunderstood my purpose: Mozilla::CA library is supported by default on all platform.

"ssl-keystore" is only there to permit usage of certificates deployed in keystore or keychain as related OS provides tools to handle such deployment.

On other OS, we don't have such usage and people still have other solutions. So don't enable "ssl-keystore" support on other OS and keep this code.

Reply:
From what I've seen, the Mozilla::CA isn't loaded because this line exit the function if OS is not macOS or Windows, but if Mozilla::CA is supported on Unix/Linux systems but this support isn't enable because the code exits the function before importing it.

Reply:
Mozilla::CA is used by LWP::UserAgent http client by default.

The purpose of `_KeyChain_or_KeyStore_Export()` function is to load keystore or keychain certificates and provide them via "SSL_ca" param of IO::Socket::SSL ssl_opts() method. Using this params also means default Mozilla::CA won't be loaded by IO::Socket::SSL. So this means we need to add them ourselves to SSL_ca param.

Reply:
I see, thank you for your clarification.